task_resource/views/project_project_view.xml

```diff
@@ -6,31 +6,25 @@
         <field name="inherit_id" ref="project.edit_project"/>
```

Comment:
Remove all the view because the purpose of this inherited view was override all the original field of the native project module to add the "attrs" atribute to that fields, so, if the field "state" gone this view is not necessary.

Thanks :)